src/opml-generator.js

```diff
@@ -0,0 +1,17 @@
+const opml = require('opml-generator');
+
+const header = {
+  title: 'OPML Feeds',
+  dateCreated: new Date(2020, 2, 9),
+  ownerName: 'Jordan',
+};
+
+const outlines = {
+  text: 'txt',
+  title: 'My Open Source Experience',
+  type: 'rss',
+  xmlUrl: 'https://jrdnlxopensource.blogspot.com/feeds/posts/default/-/open-source',
+  htmlUrl: 'https://jrdnlxopensource.blogspot.com/',
+};
+
+module.exports = opml(header, outlines);
```

Comment:
`outlines` is expected to be an Array vs. an Object, see https://www.npmjs.com/package/opml-generator#usage